### DISCUSSION TITLE
Audiobook playback through Music Assistant V2

### DISCUSSION THREAD

**Discussion by CunningSquirrel** [music-assistant/support#1182] [Feature requests and ideas]
Just like we have the ability to attach a Spotify account or “Plex Server Library”, it would be amazing if we could stream our audiobooks through music assistant by linking it to a local “Audiobookshelf” server we have running (mine runs in docker).

**Comment by OzGav** [comment]
I’ve never used this. Looking at the docs it seems it uses the ID3 tags to obtain its metadata. What extra value does the audiobookshelf server provide? If your audio books have ID3 tags populated then can you just add your audio book directory as a new SMB provider?

**Reply by CunningSquirrel** [reply]
Well it’s essentially a self hosted Audible server. So it keeps track of your audiobook progress. Meaning you can stop and start your progress across multiple devices, because it uses the user account to track your progress and not the device playing the audiobook. It allows you to add a sleep timer as well for the audiobook, so the book will pause once the count down reaches 0. It really is a great server to use if you love your audiobooks. It also allows you to group books together, by series, authors etc. So it’s easier to go from book 1 to book 2 as you work your way through a series.

**Reply by OzGav** [reply]
Thanks. Audio books are on the roadmap but not high priority at the moment as the devs are working to get MA out of beta. Keep an eye on the discussions though and upvote any related to this.

**Comment by OzGav** [comment]
Duplicate of https://github.com/orgs/music-assistant/discussions/429
